CMakeLists.txt

```diff
@@ -814,6 +814,7 @@ if(_BUILD_SHARED_LIBRARY)
 endif()
 
 if(OCOS_USE_CUDA)
+  target_include_directories(ocos_operators PUBLIC ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
```

Comment:
Interesting, do you know why this missing line doesn't cause any trouble in Windows platform?

Reply:
I'm also curious. would expect the usual cmake public header propagation to handle this given the target_link_libraries above:
https://github.com/microsoft/onnxruntime-extensions/blob/e10b4118b6f5584cfa94d1429f5b386dd91f7014/CMakeLists.txt#L775

does the ordering of these lines with the target_link_libraries call matter?

Reply:
I don't know the reason now. Maybe it's caused by different generators.

Reply:
@edgchen1 @mszhanyi , I guess the reason could be CUDA windows installation already put this path into MSCV default path. but I didn't verify that.